DeveloperGame/DeveloperGame/DeveloperGameClass.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+
+public class DeveloperGameClass
+{
+    public DeveloperGameClass()
+    {
+        // To do.
+    }
+
+    public void Run()
+    {
+        PrintIntro();
+
+        while (true)
+        {
+            char userMenuChoice = GetUserMenuChoice();
+
+            if (UserChoseToPlay(userMenuChoice))
+            {
+                Play();
+            }
+            if (UserChoseToSeeLeaderBoard(userMenuChoice))
```

Comment:
While it is true that UserChoseToPlay() and UserChoseToSeeLeaderBoard() should never be true on the same character, as another programmer I had to go and look at these functions to see that they were mutually exclusive.

To make it easier to read exclusive choices, please use if, else if, and else so we know only one can happen just by looking at this code, without having to step in.

Reply:
I fixed this issue.